setup.cfg

```diff
@@ -22,6 +22,7 @@ project-urls =
     Documentation = https://python-relion.readthedocs.io/
     GitHub = https://github.com/DiamondLightSource/python-relion
     Bug-Tracker = https://github.com/DiamondLightSource/python-relion/issues
+zocalo.wrappers = dlstbx.wrapper.relion:RelionWrapper
```

Comment:
this needs to go into the `[options.entry_points]` block below and should read
```
zocalo.wrappers =
    relion = relion.zocalo.wrapper:RelionWrapper
```

The python file then also needs moving into `/relion/zocalo/` and renamed to `wrapper.py`

Reply:
This PR at the moment adds a file `/zocalo/relion-results-wrapper.py` which is outside of `/src/relion/` and so is not importable and can't be used for entry points. I would suggest moving that to `/src/relion/zocalo/wrapper.py` (missed out the `/src/` bit in my original comment), at which point it will be importable under the name that I wrote in the comment above.

You are right that there is a script in `/zocalo/wrapper.py`. Ultimately that one should disappear and be merged together with the file that this PR adds.